frontend/src/app/main/ui/workspace/sidebar/options/menus/component.scss

```diff
@@ -716,22 +716,19 @@
   gap: var(--sp-xs);
 }
 
-.variant-property-name-bg {
-  border-radius: $br-8;
-  background-color: var(--assets-item-background-color);
-}
-
 .variant-property-name {
   color: var(--color-foreground-primary);
-
-  width: $s-104;
+  height: var(--sp-xxxl);
 
   display: flex;
   align-items: center;
   justify-content: center;
-  height: var(--sp-xxxl);
 
   overflow: hidden;
   text-overflow: ellipsis;
   white-space: nowrap;
 }
+
+.variant-property-name-width {
```

Comment:
I don't think that we should use "width" as part of the name of a class...

Maybe call it `:variant-property-name-wrapper` or something similar...

Also, I will keep the width also on `variant-property-name`, and use one or the other